content/docs/guides/reflex.md

```diff
@@ -0,0 +1,224 @@
+
+```python
+import os
+
+DB_URL = os.getenv("NEON_DATABASE_URL")
+
+config = rx.Config(
+    app_name="new_project",
+    db_url=os.getenv("DB_URL"),
+    frontend_port=3001,
+)
+```
+
+</Admonition>
+
+3. Save the changes to the `rxconfig.py` file.
+
+Now, you can run the Reflex server and start building your Python full-stack application with Reflex and Neon.
+
+## Creating a data model
+
+To create a data model in Reflex, you can define a Python class that represents the data structure. Reflex uses [sqlmodel](https://sqlmodel.tiangolo.com/) to provide a built-in ORM wrapping [SQLAlchemy](/docs/guides/sqlalchemy).
+
+
+For example, you can create a `Customer` model as follows:
+
+```python
+# <new_project>/models.py
+
+import reflex as rx
+
+class Customer(rx.Model, table=True):
+    """The customer model."""
+
+    name: str
+    email: str
+    phone: str
+    address: str
+
+```
+
+This code defines a `Customer` model with fields for `name`, `email`, `phone`, and `address`. The `table=True` argument tells Reflex to create a table in the database for this class.
+
+You can then use this model to interact with the database and perform CRUD operations on the `Customer` data.
+
+
+Creating the table with the model:
+
+```bash
+reflex db create
+```
+
+This command creates the table in the database based on the model definition using an alembic migration.
+
+Now you can use the `Customer` model to interact with the database and perform CRUD operations on the `Customer` data.
+
+For example, you can add a new customer to the database as follows:
+
+```python
+with rx.session() as session:
+    session.add(
+        User(
+            name="Alice",
+            email="user@test.com",
+            phone="1234567890",
+            address="123 Main St",
+        )
+    )
+    session.commit()
+```
+
+This code creates a new `Customer` object and adds it to the database using a session. The `session.commit()` method saves the changes to the database. If you change the table schema, you can run the following command to update the database:
+
+```bash
+reflex db makemigrations --message '<describe what changed>'
+```
+
+This command generates a new migration file that describes the changes to the database schema. You can then apply the migration to the database with the following command:
+
+```bash
+reflex db migrate
+```
+
+This command applies the migration to the database, updating the schema to match the model definition.
+
+
+
+## Create a Data Viewer Dashboard in Reflex with Neon
+
+Learn how to use Reflex with Neon Postgres in the below project to create a an interactive Data Viewer Dashboard application.
+
+<DetailIconCards>
+
+<a href="https://github.com/reflex-dev/data-viewer" description="Reflex with Neon Postgres" icon="github">Reflex Data Viewer app</a>
```

Comment:
I would change this to a newer better App that we have developed that is much better than this old one: https://github.com/reflex-dev/reflex-examples/tree/main/customer_data_app.

It will be coming out later this week, when we push all the updates, and I can send over the deployed app as well.